### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by xenide**
Anything else that needs to be documented? 

For the DAO treasury and investor + team vesting period, I'll update when I've written the deploy vester script. 

If not, we'll just update the mainnet and testnet addresses in the table after deployment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+
+All three of these components work together with their own sphere of influence. The VEX token essentially functions as a voter registration tool (and as a tradable VIP180 token), the GovernorAlpha acts as a polling location- the place where voting happens, and the Timelock acts as a loading bay that holds decision for a set amount of time before executing them on the network. 
+
+Collected fees from Vexchange V2 will also be held in this smart contract. 
+
+### Parameters set for Vexchange
+
+- `VEX::totalSupply`: set at 100 million
+- `GovernorAlpha::quorumVotes`: set at 3% of initial supply of VEX
+- `GovernorAlpha::proposalThreshold`: set at 0.1% of initial supply of VEX
```
Comment (OliverNChalk) [inline README.md]: `initial supply` -> `configured total supply`? Currently sounds like its talking about day 1 circulating supply

File: README.md [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+
+### **GovernorAlpha**
+
+The GovernorAlpha contract is the contract that does the actual "governance" part of the ecosystem. There are a number of hard-coded parameters that decide the functionality of governance, and the contract itself is the tool by which proposals are proposed, voted upon, and transferred to a timelock to be executed. The logic for secure voting is handled here. 
+
+### **Timelock**
+
+The final component of the system is a Timelock. Timelock contracts essentially "delay" the execution of transactions to give the community a chance for a "sanity check" to be  run over the outcome of a vote. It's important if a last minute bug is found in the system and it needs to be caught before a transaction is implemented.
```
Comment (OliverNChalk) [inline README.md]: `to be  run over` has a double space between `be` and `run`

File: README.md [hunk 3]
```diff
@@ -14,6 +47,23 @@ To do that, we first do `npm run deployGovernance [mainnet|testnet]`. If the dep
...
+| GovernorAlpha  |                                          |
+
+
+**Testnet**
+| Contract  | Address                                    |
+| ---       | ---                                        |
+| VEX      |  |
+| Timelock |  |
+| GovernorAlpha  |  |
+
```
Comment (OliverNChalk) [inline README.md]: Can probably align these tables since we will need to add addresses a decent bit later

**Review by OliverNChalk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xenide) [reply README.md]: Configured total supply gives me the feeling that total supply is fixed. How about initial total supply?

File: README.md (hunk 2 above)
Comment (xenide) [reply README.md]: thanks, will fix

File: README.md (hunk 3 above)
Comment (xenide) [reply README.md]: will fix

File: README.md (hunk 1 above)
Comment (OliverNChalk) [reply README.md]: I mean it is, uniswap has mint, compound has mint, ethereum has mint, bitcoin has mint. You still say they have total supplies, at least that's what coingecko is telling me

**Review by OliverNChalk — APPROVED** [review]

**Review by leodsoft — APPROVED** [review]

**Pull request merged**